models/intermediate/int__location.sql

```diff
@@ -33,7 +33,7 @@ SELECT
     , address_1
     , city
     , state
-    , zip
+    , {{ dbt.replace("zip", "'-'", "''") }} AS zip
```

Comment:
OMOP requires zipcodes to be at most 9 digits; Synthea was adding an unnecessary dash to zips.